stock_move_common_dest/tests/test_move_common_dest.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.tests import SavepointCase
+
+
+class TestCommonMoveDest(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.partner_delta = cls.env.ref("base.res_partner_4")
+        cls.warehouse = cls.env.ref("stock.warehouse0")
+        cls.warehouse.write({"delivery_steps": "pick_pack_ship"})
+        cls.customers_location = cls.env.ref("stock.stock_location_customers")
+        cls.output_location = cls.env.ref("stock.stock_location_output")
+        cls.packing_location = cls.env.ref("stock.location_pack_zone")
+        cls.stock_shelf_location = cls.env.ref("stock.stock_location_components")
+        cls.stock_shelf_2_location = cls.env.ref("stock.stock_location_14")
+
+        cls.out_type = cls.warehouse.out_type_id
+        cls.pack_type = cls.warehouse.pack_type_id
+        cls.pick_type = cls.warehouse.pick_type_id
+
+        cls.product_1 = cls.env["product.product"].create(
+            {"name": "Product 1", "type": "product"}
+        )
+        cls.product_2 = cls.env["product.product"].create(
+            {"name": "Product 2", "type": "product"}
+        )
+
+        cls.procurement_group_1 = cls.env["procurement.group"].create(
+            {"name": "Test 1"}
+        )
+
+    def _init_inventory(self):
+        # Product 1 on shelf 1
+        # Product 2 on shelf 2
+        inventory = self.env["stock.inventory"].create({"name": "Test init"})
+        inventory.action_start()
+        product_location_list = [
+            (self.product_1, self.stock_shelf_location),
+            (self.product_2, self.stock_shelf_2_location),
+        ]
+        lines_vals = list()
```

Comment:
```suggestion
```

Reply:
I modified the code to make a inventory adjust in 15.0. I migrated this module because is used by https://github.com/OCA/wms/pull/446 and this last one have multiple dependencies.